outputs.tf

```diff
@@ -1,11 +1,11 @@
 output "instances_role_arn" {
   description = "ARN of the IAM role of the EC2 instances"
-  value       = aws_iam_role.this.arn
+  value       = aws_iam_role.this.*.arn
 }
 
 output "instances_role_name" {
-  description = "Name of the IAM role of the EC2 instances"
-  value       = aws_iam_role.this.name
+  description = "Name of the IAM role of the EC2 instances. Will only be populated if the IAM role is created by this module"
```

Comment:
Do you not need to add this extra note to `instances_role_arn` as well?

Reply:
Of course. 🤦‍♂️